Custom organization invitations flow '__clerk_status' query parameter (#410)
clerkinc/clerk-docs: docs/organizations/inviting-users.mdx

    </div>
  );
}
```

### Custom redirect url

> Using Clerk's BE SDK, when creating an organization invitation, you can specify a custom redirect URL. When users click on organization invitation link, they get redirected to that URL after the ticket has been verified. The URL will contain two important query parameters added by Clerk, `__clerk_ticket` and `__clerk_status`.

Reviewer: ```suggestion
When creating an organization invitation and using Clerk's Next.js, Remix, or Backend SDKs you can specify a custom redirect URL. After users click on organization invitation link and the ticket is verified, they will get redirected to that URL. The URL will contain two important query parameters added by Clerk, `__clerk_ticket` and `__clerk_status`.
```

Author: Updated!

> The `__clerk_ticket` query parameter will hold the actual ticket token, which can be used during sign in and sign up flows in order to complete the organization invitation flow.

Author: Updated!

> The `__clerk_status` query parameter is the outcome of the ticket verification and can take three values:

Reviewer: ```suggestion
The `__clerk_status` query parameter is the outcome of the ticket verification and will contain one of three values:
```

> - `sign_in` which means that the user already exists in your application. You should create a ticket sign-in in order to complete the flow.

Reviewer: ```suggestion
- `sign_in` indicates the user already exists in your application. You should create a sign-in ticket in order to complete the flow.
```

> - `sign_up` which means that the user doesn't already exist in your application. You should create a ticket sign-up in order to complete the flow.

Reviewer: ```suggestion
- `sign_up` indicates the user doesn't already exist in your application. You should create a sign-up ticket in order to complete the flow.
```

Author: Updated!

> - `complete` which means that the user already exists and it was signed in. That means that the flow has been completed and no further actions are required.

Reviewer: ```suggestion
- `complete` indicates the user already exists and was signed in. The flow has been completed and no further actions are required.
```

Author: Updated!
